src/main/java/com/google/step/finscholar/data/Scholarship.java

```diff
@@ -91,47 +91,47 @@ public Scholarship(String name, UUID uuid, List<UUID> schoolsList, String URL) {
   }
 
   /**
-   * The getter of the list of finantial background requirments.
-   * @return The list of finantial requirements.
+   * The getter of the list of financial background requirements.
+   * @return The list of financial requirements.
    */
   public List<String> getFinancialRequirements() {
     return financialRequirements;
   }
 
   /**
-   * The getter for gender restriction of the scholarship.
-   * @return The list of gender requirement.
+   * The getter for gender restrictions of the scholarship.
+   * @return The list of gender requirements.
    */
   public List<String> getGenderRequirements() {
     return genderRequirements;
   }
 
   /**
-   * The getter for geographic restriction of the scholarship;
-   * @return The list of location requirement.
+   * The getter for geographic restrictions of the scholarship;
+   * @return The list of location requirements.
    */
   public List<String> getLocationRequirements() {
     return locationRequirements;
   }
 
   /**
-   * The getter for nationality restriction of the scholarship;
+   * The getter for nationality restrictions of the scholarship;
    * @return The list of national origin requirements.
    */
   public List<String> getNationalOriginRequirements() {
     return nationalOriginRequirements;
   }
 
   /**
-   * The getter for the number of year available to one student.
+   * The getter of the number of years the scholarship is offered to one student.
    * @return The duraton of the scholarship in year.
```

Comment:
Fix typos.

Reply:
Duration is misspelled (missing i between t and o). Also maybe make years plural?

Reply:
Sorry github being confusing. I saw you fix it. :)